app/javascript/src/components/Dashboard/Contacts/Table/constants.jsx

```diff
@@ -15,7 +15,7 @@ export const CONTACTS = CONTACTS_RANGE.map(contact => ({
   createdAt: "Feb, 5, 2021",
 }));
 
-export const COLUMN_HEADERS = [
+export const COLUMN_HEADERS = ({ setIsDeleteOpen }) => [
```

Comment:
I think it would be better to move it into `utils.jsx` prefixed with `build` keyword. Refer https://academy.bigbinary.com/learn-react/react-best-practices/naming-conventions-in-react#when-to-use-the-render-keyword

Reply:
Fixed in [9d26873](https://github.com/vysakh-poulose/neeto-ui-challenge-by-vysakh-poulose/pull/16/commits/9d2687384782142f7195f006c99d2b6189262a8d)